docs/deprecations.md

```diff
@@ -36,3 +38,11 @@ See [TEP-0074](https://github.com/tektoncd/community/blob/main/teps/0074-depreca
 - The [kubeconfigwriter](https://github.com/tektoncd/pipeline/blob/release-v0.43.x/pkg/apis/pipeline/images.go#L36) and [imagedigestexporter](https://github.com/tektoncd/pipeline/blob/release-v0.43.x/pkg/apis/pipeline/images.go#L46) images, used by the Cluster PipelineResource and Image PipelineResource, respectively
 
 - The [`tekton_pipelines_controller_cloudevent_count`](https://github.com/tektoncd/pipeline/blob/main/docs/metrics.md) metric
+
+## Removed features
+
+The features listed below have been removed but may still be supported in releases that have not reached their EOL.
+
+| Removed Feature                                                                                                                                                                                                   | Removal Pull Request  | Latest release with support | EOL
+|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------|-------------------------------------|
+| [The `PipelineRun.Status.TaskRuns` and `PipelineRun.Status.Runs` fields and the `embedded-status` feature flag along with their functionalities have been deprecated since v0.45.](https://github.com/tektoncd/community/blob/main/teps/0100-embedded-taskruns-and-runs-status-in-pipelineruns.md) | [[TEP100] Remove Taskruns and Runs Fields for PipelineRunStatus](https://github.com/tektoncd/pipeline/pull/6099) | v0.35.0 | Jan 25, 2023 |
```

Comment:
This says the feature was deprecated in v0.45 - was this deprecated or removed in 0.45? If it was deprecated in v0.45, the last supported release being v0.35 feels off

Reply:
Thanks Dibyo, I changed it to tombestoned if that makes sense